.github/workflows/test_integration_adapter.yml

```diff
@@ -32,6 +32,7 @@ jobs:
           - snowflake
           - redshift
           - trino
+          # - sqlserver # does not support dbt 1.4 yet
```

Comment:
OK, this is interesting for me. We deploy latest from this branch. Should this be pointing to a [1.3 branch](https://github.com/fal-ai/fal/tree/release-adapter-1.3)?

Reply:
Or just wait for 1.4?

Reply:
I think we can just wait.